CHARTER.md

```diff
@@ -1,51 +1,61 @@
+Arcalot is a **free and open source collection of projects and will remain so**. Our main mission is to improve the way workloads are run, how their data is consumed and forwarded, and to simplify analyzing the output of workloads.
 
-This charter covers **important processes and decisions that impact the whole organization, direction, and goals**. Small decisions like coding style per repository or similar details can be decided among the repository maintainers.
+This charter covers **important processes and decisions that impact the whole organization, its direction, and its goals**. Small decisions like coding style per repository or similar details can be decided among the repository maintainers.
 
 ## Arcalot Round Table
 
-The Arcalot Round Table *(ART)* consists of:
-* 4 chairs with equal rights who
+The Arcalot Round Table (ART) consists of:
+* Four chairs with equal rights who
   - Enforce the [Code of Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md)
   - Have administrative rights over the [GitHub org](https://github.com/arcalot), communication channels, and other community-owned resources
-  - Publicly record decisions made by the ART in the [art-decisions](art-decisions) folder
-* Individuals who want to join the ART and pass a review of their PR sent to [ART_MEMBERS.md](ART_MEMBERS.md), with approval depending on any one or more of the below requirements:
-  - They contribute with code, documentation, artwork, articles, videos, talks, or similar additions to the Arcalot ecosystem
-  - They are an active user of Arcalot projects (end user, product builder, researcher, etc. - please describe your use case in the PR, you do not have to mention any company names or details)
-  - They are actively helping community members on any of the official Arcalot channels
+  - Publicly record decisions made by the ART in the [`art-decisions`](art-decisions) folder
+* Individuals who
+  - Contribute with code, documentation, artwork, articles, videos, talks, or similar additions to the Arcalot community
+  - Are users of Arcalot projects (end users, product builders, researchers, etc.)
+  - Help community members on any of the official Arcalot channels
 
-### Termination of ART membership
+### ART Participation
 
-An Arcalot Round Table member *(including chairs)* that does not participate in round table matters (voting) *without reason and notice* for 3 times in a row leaves the round table automatically as we can assume they are not interested in being a member of the ART anymore. This ensures that we do not have a corpus of inactive members with voting rights.
+If you wish to join the ART, please submit a pull request adding yourself to the [`ART_MEMBERS.md`](ART_MEMBERS.md) file. Please describe your use case in the pull request comments; you do not need to mention any company names or details. Joining is finalized upon two pull request approvals by active chairs and the merging of the pull request.
 
```

Comment:
I'd prefer to make sure that "joining" has a minimal barrier to entry. Two approvals aligns with the existing branch protection rules, making this administratively simple. It seems like cause for rejecting a new member should be extremely rare, but if we were to the point of managing new members on a weekly or daily basis, I could see a risk begin to arise. I feel like unanimous approval from the chairs is needlessly burdensome and could delay members from joining for weeks in the case of an unavailable chair. If we did it by share of those chairs available to vote, so unanimous consent as you describe, we would need to set a max time and wait for all approvals or a timeout simply to allow someone to join the community.

Reply:
Hmm; what, functionally, does "joining the ART" mean? Anyone can raise a PR to add themselves to a potentially really long and unsorted list of names: so, when that's merged (and who handles all the rebases when multiple additions inevitably conflict?), what does this actually *mean* for the operation of the ART?

It doesn't add the name to a GitHub team or email list -- someone would have to do that manually. It doesn't give the new member any specific rights anywhere -- someone would have to do that manually. We could in theory write scripts to validate authors (email, PR, review, etc) against this list, but it wouldn't happen automatically.

Reply:
Yes, there are right now several manual processes that a chair will have the burden of administering any time someone joins or leaves, including modifying group membership in the github org, managing e-mail groups, and such. We'd certainly need to automate those things if we hit a high enough user churn rate. But I'm not sure there is anything more about this that needs to be called out in the charter.

Reply:
OK, @dustinblack, so your concern is that using unanimous consent would slow the process of joining; understood.  (Although, the fact that a new member has to wait a significant period of time before they can vote makes me think that having to wait a short time for approval shouldn't be a big deal...if necessary, we can make the other timeouts start at _application_ time instead of approval time.)

However, my other questions stand:  what if, after an initial chair approves an application, the second chair's response is negative?...what if the third response is also negative?...does the approval still happen if the fourth chair then approves?

Reply:
> However, my other questions stand: what if, after an initial chair approves an application, the second chair's response is negative?...what if the third response is also negative?...does the approval still happen if the fourth chair then approves?

That may be an entirely valid and acceptable result. The chairs should not have undue influence over who joins the community, and the effect of the current language is a simple 50% approval by the chairs.

Reply:
OK...I was just checking....  🙂